app/models/appeals/AppealSubmissionRequest.scala

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.appeals
+
+import play.api.libs.json.{JsObject, JsValue, Json, Writes}
+import java.time.{LocalDateTime, ZoneOffset}
+
+case class AppealSubmissionRequest (
+  taxRegime: String,
+  penaltyId: String,
+  dateOfAppeal: LocalDateTime,
+  isLPP: Boolean,
+  appealSubmittedBy: String,
+  appealInformation: AppealInformation,
+  agentDetails: Option[AgentDetails]
+){
+  val sourceSystem: String = "MDTP"
+  val appealLevel: String = "01"
+}
+
+object AppealSubmissionRequest {
+  def apply(appealSubmission: AppealSubmission, penaltyId:String): AppealSubmissionRequest = {
+    AppealSubmissionRequest(
+      taxRegime = appealSubmission.taxRegime,
+      penaltyId = penaltyId,
+      dateOfAppeal = appealSubmission.dateOfAppeal,
+      isLPP = appealSubmission.isLPP,
+      appealSubmittedBy = appealSubmission.appealSubmittedBy,
+      appealInformation = appealSubmission.appealInformation,
+      agentDetails = appealSubmission.agentDetails
+    )
+  }
+
+  def parseAppealInformationToJson(payload: AppealInformation): JsValue = {
+    val value = payload.reasonableExcuse match {
+      case "bereavement" =>
+        Json.toJson(payload.asInstanceOf[BereavementAppealInformation])(BereavementAppealInformation.bereavementAppealWrites)
+      case "crime" =>
+        Json.toJson(payload.asInstanceOf[CrimeAppealInformation])(CrimeAppealInformation.crimeAppealWrites)
+      case "fireandflood" =>
```

Comment:
Is this not camelcase like the others? And is 'and' not 'or' like the object?

Reply:
The same identifier is used in the frontend as well: https://github.com/search?q=repo%3Ahmrc%2Fpenalties-appeals-frontend%20fireandflood&type=code

Is it worth having a ticket to check the dependencies and refactor them?